composer.json

```diff
@@ -29,6 +29,12 @@
         "process-timeout": 1800
     },
     "scripts": {
+         "post-install-cmd": [
+            "php -r \"if(!file_exists('config/console-local.php')){copy('config/templates/console-local.php', 'config/console-local.php');}\"",
```

Comment:
I'd prefer moving these operations to the separate method of `yii\composer\Installer` class.